apps/ReconstructMesh/ReconstructMesh.cpp

```diff
@@ -167,12 +169,12 @@ bool Initialize(size_t argc, LPCTSTR* argv)
 	// validate input
 	Util::ensureValidPath(OPT::strInputFileName);
 	Util::ensureUnifySlash(OPT::strInputFileName);
-	if (OPT::vm.count("help") || OPT::strInputFileName.IsEmpty()) {
+	if (OPT::vm.count("help") || (OPT::strInputFileName.IsEmpty() && OPT::strMeshFileName.IsEmpty())) {
```

Comment:
I'd implement this differently:
pass the mesh file as the input instead as using --mesh-file param, and at the scene loading time you have an import option which if set on true, it allows to load a mesh too instead of only MVS

Reply:
Hey @cdcseacave, thanks for the feedback. Could you expand more on this one?

>  at the scene loading time you have an import option which if set on true

Is this an additional flag? Could one just check for the file extension of the input too? (If it's MVS, load scene, if it's OBJ/PLY, load mesh).